src/structures/game_round.ts

```diff
@@ -362,50 +367,79 @@ export default class GameRound extends Round {
 
     /**
      * @param correctGuesses - The number of correct guesses
-     * Marks button guesses as correct or incorrect in a multiple choice game
+     * @param showCorrectAnswer - Whether to show the correct answer
+     * Disables buttons in a multiple choice game
      */
-    async interactionMarkAnswers(correctGuesses: number): Promise<void> {
+    async interactionMarkAnswers(
+        correctGuesses: number,
+        showCorrectAnswer: boolean,
+    ): Promise<void> {
         if (!this.interactionMessage) return;
-        try {
-            await this.interactionMessage.edit({
-                embeds: this.interactionMessage.embeds,
-                components: this.interactionComponents.map((x) => ({
-                    type: 1,
-                    components: x.components.map((y) => {
-                        const z = y as Eris.InteractionButton;
+        const actionRows: Eris.ActionRow[] = this.interactionComponents.map(
+            (actionRow) => ({
+                type: 1,
+                components: actionRow.components.map(
+                    (button: Eris.InteractionButton) => {
+                        if (
+                            Object.values(ClipAction).includes(
+                                button.custom_id as ClipAction,
+                            )
+                        ) {
+                            // TODO heart here
```

Comment:
is this TODO still valid

Reply:
I plan to get this resolved in https://github.com/Brainicism/KMQ_Discord/pull/2031